AU2/plugins/custom_plugins/PolicePlugin.py

```diff
@@ -88,122 +93,205 @@ def __init__(self):
                 "Police -> Set Ranks",
                 self.ask_set_ranks,
                 self.answer_set_ranks
+            ),
+            ConfigExport(
+                "PolicePlugin_toggle_auto",
+                "Police -> Toggle Auto Ranking",
+                lambda *args: [],
+                self.answer_toggle_auto
+            ),
+            ConfigExport(
+                "PolicePlugin_toggle_manual",
+                "Police -> Toggle Manual Ranking",
+                lambda *args: [],
+                self.answer_toggle_manual
+            ),
+            ConfigExport(
+                "PolicePlugin_toggle_police_kills_rankup",
+                "Police -> Toggle Ranking up on Police Kills",
+                lambda *args: [],
+                self.answer_toggle_police_kills_rankup
             )
         ]
 
-        self.ranks = GENERIC_STATE_DATABASE.arb_state.setdefault(self.identifier, DEFAULT_RANKS)
+    # I've done the database stuff in such a way that configs are only stored if changed; otherwise the hardcoded default configs are used.
```

Comment:
This is sensible, and is good design. It saves us file space on our generally limited SRCF provision!